doc/api/index.rst

```diff
@@ -58,6 +58,13 @@ Saving and displaying the figure:
     Figure.show
     Figure.psconvert
 
+Configuring the display settings:
+
+.. autosummary::
+    :toctree: generated
+
+    set_display
```

Comment:
I'm thinking whether to put `set_display` alongside `config` and have a heading like "Advanced configuration" or something like that.